web/src/components/Utils/Utils.js

```diff
@@ -0,0 +1,3 @@
+export const HandleReplace = (string, from, to) => {
```

Comment:
As already discussed on slack and in previous comments, `handle` and `on` prefixes are better to be used for the events. Like user interactions, mouse events, keyboard events, and so on. Replacing a string is not an event. So none of these prefixes should be used here. Nor `on` neither `handle`. The functions also should be camel case, better not to use pascal case for functions.

Reply:
Yes Amir, we already discussed this in slack. Previously I removed the prefix `on` and now changed it into camel case.